enterprise/utils.py

```diff
@@ -49,19 +49,7 @@ def __init__(self, *args, **kwargs):
         Log a warning and initialize the exception.
         """
         LOGGER.warning('edx-enterprise unexpectedly failed as if not installed in an OpenEdX platform')
-        super(NotConnectedToEdX, self).__init__(*args, **kwargs)
-
-
-class NotConnectedToOpenEdX(Exception):
-    """
-    Exception to raise when we weren't able to import needed resources.
-
-    This is raised when we try to use the resources, rather than on
-    import, so that during testing we can load the module
-    and mock out the resources we need.
```

Comment:
Yea, I'm in favor of keeping it in this PR, I just wanted to call it out since it wasn't essential to the purpose of the PR.